scripts/scil_compute_msmt_fodf.py

```diff
@@ -0,0 +1,239 @@
+
+    # Computing msmt-CSD
+    msmt_model = MultiShellDeconvModel(gtab, msmt_response,
+                                       reg_sphere=reg_sphere,
+                                       sh_order=sh_order)
+
+    # Computing msmt-CSD fit
+    msmt_fit = fit_from_model(msmt_model, data,
+                              mask=mask, nbr_processes=args.nbr_processes)
+
+    shm_coeff = msmt_fit.all_shm_coeff
+
+    nan_count = len(np.argwhere(np.isnan(shm_coeff[..., 0])))
+    voxel_count = np.prod(shm_coeff.shape[:-1])
+
+    if nan_count / voxel_count >= 0.05:
+        msg = """There are {} voxels out of {} that could not be solved by
+        the solver, reaching a critical amount of voxels. Make sure to tune the
+        response functions properly, as the solving process is very sensitive
+        to it. Proceeding to fill the problematic voxels by 0.
+        """
+        logging.warning(msg.format(nan_count, voxel_count))
+    elif nan_count  > 0:
+        msg = """There are {} voxels out of {} that could not be solved by
+        the solver. Make sure to tune the response functions properly, as the
+        solving process is very sensitive to it. Proceeding to fill the
+        problematic voxels by 0.
+        """
+        logging.warning(msg.format(nan_count, voxel_count))
+
+    shm_coeff = np.where(np.isnan(shm_coeff), 0, shm_coeff)
+
+    # Saving results
+    if args.wm_out_fODF:
+        wm_coeff = shm_coeff[..., 2:]
+        if args.sh_basis == 'tournier07':
+            wm_coeff = convert_sh_basis(wm_coeff, reg_sphere, mask=mask,
+                                         nbr_processes=args.nbr_processes)
+        nib.save(nib.Nifti1Image(wm_coeff.astype(np.float32),
+                                 vol.affine), args.wm_out_fODF)
+
+    if args.gm_out_fODF:
+        gm_coeff = shm_coeff[..., 1]
+        if args.sh_basis == 'tournier07':
+            gm_coeff = gm_coeff.reshape(gm_coeff.shape + (1,))
+            gm_coeff = convert_sh_basis(gm_coeff, reg_sphere, mask=mask,
+                                         nbr_processes=args.nbr_processes)
+        nib.save(nib.Nifti1Image(gm_coeff.astype(np.float32),
+                                 vol.affine), args.gm_out_fODF)
+
+    if args.csf_out_fODF:
+        csf_coeff = shm_coeff[..., 0]
+        if args.sh_basis == 'tournier07':
+            csf_coeff = csf_coeff.reshape(csf_coeff.shape + (1,))
+            csf_coeff = convert_sh_basis(csf_coeff, reg_sphere, mask=mask,
+                                         nbr_processes=args.nbr_processes)
+        nib.save(nib.Nifti1Image(csf_coeff.astype(np.float32),
+                                 vol.affine), args.csf_out_fODF)
+
+    if args.vf:
+        nib.save(nib.Nifti1Image(msmt_fit.volume_fractions.astype(np.float32),
+                                 vol.affine), args.vf)
+
+    if args.vf_rgb:
+        vf = msmt_fit.volume_fractions
+        vf_rgb = (vf + np.min(vf)) / (np.max(vf)-np.min(vf)) * 255
```

Comment:
If I am not mistaken, this will result in some values outside the range `[0, 255]` whenever `np.min(vf)` is different than 0. I think rewriting the equation with a minus sign would fix the issue:
```suggestion
        vf_rgb = (vf - np.min(vf)) / (np.max(vf)-np.min(vf)) * 255
```
However, correct me if I'm wrong, I don't think the minimum value for volume fractions can ever be negative because of the positivity constraint, and it seems more natural to me for the color `[0, 0, 0]` (black) to always correspond to volume fractions of `[0, 0, 0]` rather than to the minimum volume fraction. In that case, we could simply write:
```suggestion
        vf_rgb = vf / np.max(vf) * 255
```